Shared/Samples/Show viewshed from geoelement in scene/ShowViewshedFromGeoelementInSceneView.swift

```diff
@@ -0,0 +1,194 @@
+            graphicsOverlay.renderer = renderer3D
+            
+            return graphicsOverlay
+        }()
+        
+        /// The analysis overlay for the tank's viewshed.
+        let analysisOverlay = AnalysisOverlay()
+        
+        /// The graphic for the tank.
+        let tankGraphic: Graphic = {
+            let tankSymbol = ModelSceneSymbol(url: .bradleyTank, scale: 10.0)
+            tankSymbol.heading = 90.0
+            tankSymbol.anchorPosition = .bottom
+            let tankGraphic = Graphic(
+                geometry: Point(x: -4.506390, y: 48.385624, spatialReference: .wgs84),
+                attributes: ["HEADING": 0.0],
+                symbol: tankSymbol
+            )
+            return tankGraphic
+        }()
+        
+        /// The point for the tank to move toward.
+        var waypoint: Point!
+        
+        /// The timer for the moving tank animation.
+        var animationTimer: Timer!
+        
+        init() {
+            // Create camera controller.
+            cameraController = OrbitGeoElementCameraController(
+                target: tankGraphic,
+                distance: 200.0
+            )
+            
+            // Add tank graphic to graphics overlay.
+            graphicsOverlay.addGraphic(tankGraphic)
+            
+            // Create a viewshed to attach to the tank.
+            let geoElementViewshed = GeoElementViewshed(
+                geoElement: tankGraphic,
+                horizontalAngle: 90.0,
+                verticalAngle: 40.0,
+                headingOffset: 0.0,
+                pitchOffset: 0.0,
+                minDistance: 0.1,
+                maxDistance: 250.0
+            )
+            
+            // Offset viewshed observer location to top of tank.
+            geoElementViewshed.offsetZ = 3.0
+            
+            // Add the viewshed to the analysisOverlay to add to the scene.
+            analysisOverlay.addAnalysis(geoElementViewshed)
+        }
+        
+        /// Animate the tank moving from its current point to the waypoint.
+        func animate() {
+            // Get point from the current tank position.
+            guard let tankLocation = tankGraphic.geometry as? Point else { return }
+            
+            // Get the distance from the current tank location to the waypoint.
+            guard let distanceResult = GeometryEngine.geodeticDistance(
+                from: tankLocation,
+                to: waypoint,
+                distanceUnit: LinearUnit.meters,
+                azimuthUnit: AngularUnit.degrees,
+                curveType: .geodesic
+            ) else { return }
+            
+            // Move toward waypoint a short distance.
+            let locations = GeometryEngine.geodeticMove(
+                [tankLocation],
+                distance: 1.0,
+                distanceUnit: LinearUnit.meters,
+                azimuth: distanceResult.azimuth1.value,
+                azimuthUnit: AngularUnit.degrees,
```

Comment:
The azimuth unit should be the the unit of the azimuth measurement itself.

```suggestion
                azimuthUnit: distanceResult.azimuth1.unit.angularUnit,
```

However, that really shouldn't be necessary. Please create an API issue for adding another `geodeticMove` that takes a `distance` and `azimuth` as a measurement (`Measurement`).